packages/app-store/dailyvideo/lib/VideoApiAdapter.ts

```diff
@@ -322,7 +324,7 @@ const DailyVideoApiAdapter = (): VideoApiAdapter => {
     };
   };
 
-  async function createInstantMeeting(endTime: string, region?: RoomGeo) {
+  async function createInstantMeeting(endTime: string, bookingUid: string, region?: RoomGeo) {
```

Comment:
nit: Better to convert it to name params format. It avoids the issue we are fixing here by moving optional parameter later in the list 
```suggestion
  async function createInstantMeeting({endTime: string, bookingUid: string, region?: RoomGeo)} {
```